test/ModalSpec.js

```diff
@@ -11,6 +11,16 @@ import { render, shouldWarn } from './helpers';
 
 const $ = componentOrNode => jQuery(ReactDOM.findDOMNode(componentOrNode));
 
+
+class ErrorBoundary extends React.Component {
```

Comment:
Is this related to the `React.Children.only` test failures (#207)? If so, a separate PR might be a good idea.